selfdrive/common/util.h

```diff
@@ -182,7 +188,9 @@ class LogState {
     zmq_setsockopt(sock, ZMQ_LINGER, &timeout, sizeof(timeout));
 
     zmq_connect(sock, endpoint);
-  };
+    initialized = true;
+  }
+
   ~LogState() {
     zmq_close(sock);
```

Comment:
You can reach this code now with an uninialized `LogState`. Check for `initialized` or you'll try to close an unopened socket.

Also set `zctx` and `sock` to `nullptr` as default values just to be sure.